shadow-dom/events/event-dispatch/test-003.html

```diff
@@ -37,19 +37,29 @@
     var s = host.createShadowRoot();
 
     var div1 = d.createElement('div');
-    div1.setAttribute('style', 'height:100%; width:100%');
+    div1.setAttribute('style', 'height:50%; width:100%');
     div1.setAttribute('id', 'div1');
     s.appendChild(div1);
 
-    s.addEventListener('mouseover', A_05_05_03_T01.step_func(function(event) {
+    var div2 = d.createElement('div');
+    div2.setAttribute('style', 'height:50%; width:100%');
+    div2.setAttribute('id', 'div2');
+    s.appendChild(div2);
+
+    // Add an event listner above its shadow root. If both target and
+    // relatedTarget are in the same shadow tree under the root,
+    // retargeted target and relatedTarget will be the same node
+    // and the event must not invoke the handler.
+    host.addEventListener('mouseover', A_05_05_03_T01.step_func(function(event) {
```

Comment:
An event created by a script, e.g. document.createEvent("MouseEvents"), isn't a trusted event, I think.

> I don't think we have interoperable way to dispatch a trusted event in web platform tests.

That's the point. I'm afraid that we don't have a way to test this feature.